### PR TITLE
RSWEB-6655: derek snowflakes homepage and derek.scss sass fixes

### PR DESCRIPTION
**Pull request by SamuelRandall**
https://jira.rax.io/browse/RSWEB-6655

### CODE REVIEW

File: styleguide/_themes/derek/scss/snowflakes/_homepage.scss [hunk 1]
```diff
@@ -1,65 +1,76 @@
 .homepage-hero {
-  @extend .full-width;
-  height: 100%;
-  overflow: visible;
-  background-repeat: no-repeat;
-  top: 65px;
-  margin-bottom: 40px;
-  background-image: url("https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/backgrounds/homepage.jpg");
+  @extend %full-width;
```
Comment (writes) [inline styleguide/_themes/derek/scss/snowflakes/_homepage.scss]: nope

File: styleguide/_themes/derek/scss/snowflakes/_homepage.scss [hunk 2]
```diff
@@ -1,65 +1,76 @@
...
   border-bottom: 0;
+  height: 100%;
+  margin-bottom: 40px;
+  overflow: visible;
+  position: relative;
+  top: 65px;
+
   h2 {
-    padding-top: 20px;
+    @extend %white;
```
Comment (writes) [inline styleguide/_themes/derek/scss/snowflakes/_homepage.scss]: nope

File: styleguide/_themes/derek/scss/snowflakes/_homepage.scss [hunk 3]
```diff
@@ -1,65 +1,76 @@
...
-      hr{ border-color:$mediumteal; }
-      h3, a{ color:$mediumteal; }
+#hosting-options {
+  text-align: center;
+
+  .product {
+    p { margin-bottom: 30px; }
+
+    &:hover {
+      @extend %bg-light-gray;
```
Comment (writes) [inline styleguide/_themes/derek/scss/snowflakes/_homepage.scss]: nope

File: styleguide/_themes/derek/scss/snowflakes/_homepage.scss [hunk 4]
```diff
@@ -1,65 +1,76 @@
...
+      hr { border-color: $mediumteal; }
```
Comment (writes) [inline styleguide/_themes/derek/scss/snowflakes/_homepage.scss]: nope

File: styleguide/_themes/derek/scss/snowflakes/_homepage.scss [hunk 5]
```diff
@@ -1,65 +1,76 @@
...
 }
 
-#solutions{
-  a{ text-decoration:none; }
-  .solution:hover{
-    .border-white{
-     border-color:$mustard;
-     i{ color:$mustard; }
+#solutions {
+  a { text-decoration: none; }
```
Comment (writes) [inline styleguide/_themes/derek/scss/snowflakes/_homepage.scss]: nope

Comment (amberRrucker) [inline styleguide/_themes/derek/scss/snowflakes/_homepage.scss]: The homepage scss file is getting a rewrite today so I wouldnt worry about this one right now guys.

**Comment by SamuelRandall** [comment]
Amber is currently reworking the homepage file, changes will be made by her.

**Pull request closed without merging**